config/jobs/image-pushing/k8s-staging-csi-secrets-store.yaml

```diff
@@ -0,0 +1,37 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/secrets-store-csi-driver:
+    # The name should be changed to match the repo name above
+    - name: secrets-store-csi-driver-push-image
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        # If this is the first one for your sig, you may need to create one
+        testgrid-dashboards: sig-auth-secrets-store-csi-driver
+      decorate: true
+      # we only need to run if necessary (e.g.: the version was bumped)
+      run_if_changed: '^docker\/'
```

Comment:
This doesn't match any file in your repo

Reply:
There is a PR open for moving the files to different dir - https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/189